nexus_constructor/nexus/nexus_wrapper.py

```diff
@@ -43,6 +43,34 @@ def decode_bytes_string(nexus_string):
         return nexus_string
 
 
+def get_fields(
+    group: h5py.Group
+) -> (List[h5py.Dataset], List[h5py.Dataset], List[h5py.Group], List[h5py.Group]):
+    """
+    Return a list of fields in a given component group.
+    :param group: The hdf5 component group to check for fields
+    :return: A list of a fields, regardless of field type
+    """
+    scalar_fields = []
+    array_fields = []
+    stream_fields = []
+    link_fields = []
+    for item in group.values():
+        if isinstance(item, h5py.Dataset) and item.name.split("/")[-1] != "description":
+            if np.isscalar(item.value):
+                scalar_fields.append(item)
+                break
```

Comment:
Do you mean `continue` here? I don't think you want to completely stop the loop?